src/declaration.d

```diff
@@ -1329,6 +1333,7 @@ public:
                         // Note this prevents using foreach() over members, because the limits can change
                         sc.scopesym.members.push(v);
                 }
+
                 Expression e = new DsymbolExp(loc, v);
```

Comment:
Here is the source of wrong line number (the location where a tuple variable declared).